packages/manager/modules/pci/src/projects/project/notebooks/notebooks.routing.js

```diff
@@ -14,8 +14,8 @@ export default /* @ngInject */ ($stateProvider) => {
     resolve: {
       goToAddNotebook: /* @ngInject */ ($state, projectId) => () =>
         $state.go('pci.projects.project.notebooks.add', { projectId }),
-      notebooks: /* @ngInject */ ($q, NotebookService, projectId) =>
-        NotebookService.getNotebooks(projectId),
+      /* notebooks: /!* @ngInject *!/ ($q, NotebookService, projectId) =>
```

Comment:
We need this in the list page :)
Can you uncomment it please?

Reply:
Yes i know.
its mistake from my side.